lib/mayaUsd/render/pxrUsdMayaGL/proxyDrawOverride.cpp

```diff
@@ -64,10 +64,11 @@ UsdMayaProxyDrawOverride::UsdMayaProxyDrawOverride(const MObject& obj) :
         MHWRender::MPxDrawOverride(obj,
                                    UsdMayaProxyDrawOverride::draw
 #if MAYA_API_VERSION >= 201651
```

Comment:
If we don't care about pre-2018 maya versions, we should also remove this  conditional compilation.

Reply:
Makes sense. I can put up a separate PR for that later since it's not directly related to the changes here, and I'm sure there are a bunch of other places where we're testing for pre-2018 versions.

Reply:
LGTM.